samples/client/petstore/typescript-angular-v2/npm/tsconfig.json

```diff
@@ -5,7 +5,7 @@
         "noImplicitAny": false,
         "suppressImplicitAnyIndexErrors": true,
         "target": "es5",
-        "module": "es6",
+        "module": "commonjs",
```

Comment:
Is this change correct? Seems like it was introduced in another PR.

Reply:
@TiFu yes, that's right, I guess this pops up since I had to re-generate all typescript files.